Rakefile

```diff
@@ -12,6 +12,102 @@
+# |    |                                 | regen_individual_require_files as an enhancement. This task calls clean_individual_profiles      |
+# |    |                                 | as prereq.                                                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 7  | clean_all_profiles              | Cleans all profiles inside the rollup gem and individual gems.This task has clean_rollup_profiles|
+# |    |                                 | and clean_individual_profiles as prereqs.                                                        |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 8  | clean_rollup_profiles           | Cleans all profiles inside the rollup gem.                                                       |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 9  | clean_individual_profiles       | Cleans all profiles inside individual gems.                                                      |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 10 | regen_all_require_files         | Regenerates the require files for the rollup gem and individual gems. This task has              |
+# |    |                                 | regen_rollup_require_files and regen_individual_require_files as prereqs.                        |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 11 | regen_rollup_require_files      | Regenerates the require files for the rollup gem                                                 |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 12 | regen_individual_require_files  | Regenerates the require files for the individual gems                                            |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 13 | release                         | Release gems. This task has build as a prereq.                                                   |
+# |----|---------------------------------|--------------------------------------------------------------------------------------------------|
+# | 14 | build                           | Builds all gems. This task has clean as a prereq.                                                |
```

Comment:
similarly here, what controls what gets build should be indicated.

Reply:
build task builds for all gems.